### PR TITLE
Features/tile and popular movies ADDED MEDIA QUERIES

### PR DESCRIPTION
**Pull request by traczoskar**
### What has been changed:

- **added 2 new breakpoints to theme.js** - "tabletHorizontalMax "and "tabletVerticalMax"
- **added media queries for all components** - now every Tile should be prompt properly on almost every device.
- **changed display of a Tile from flex to grid** - 😅 You were right folks! `display: grid;` fits better and responsive for this appliance. 
- **added :hover styling for Tile components** - hope you'll like it!
- **added normalize.css link to index.html**

_I'm waiting for your feedback. Thanks!_

### CODE REVIEW

File: src/common/Tile/styled.js
Comment (wojdaSoftware) [inline src/common/Tile/styled.js:1]: Hover looks good and I think that changing to grid was the right call

**Review by wojdaSoftware — CHANGES_REQUESTED** [review]
Normalize.css removes margin from body and you haven't taken that into account. When the resolution is between 1368px and the first breakpoint content touches the window sides.

In that same case I have also noticed overflow:
image

Good work overall, just some small kinks to iron out

File: src/common/Tile/styled.js [hunk 1]
```diff
@@ -24,13 +62,23 @@ export const Description = styled.div`
...
   margin: 0;
   padding: 0;
+  line-height: 20.8px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobileMax}px) {
+    font-size: 16px;
+  }
 `;
 
 export const Year = styled.caption`
```
Comment (OlgaCybulska) [inline src/common/Tile/styled.js:84]: An error appears in the console:
image
I suggest changing the caption to h3

**Comment by traczoskar** [comment]
I did fixes, waiting for your feedback ;-)

**Review by wojdaSoftware — APPROVED** [review]

**Review by OlgaCybulska — APPROVED** [review]

**Pull request merged**